src/page-content/content-jobs.tsx

```diff
@@ -30,11 +30,12 @@ export const seo: SeoProps = {
 export const hero: HeroFeatureProps = {
   id: 'jobs.herofeatures',
   thin: true,
-  title: 'Work with us to transform',
+  title: 'Work on things that matter',
   description: (
     <>
-      At Ory, we build secure and open Auth so developers don't have to. Ory is
-      open source, cloud-native, and building at scale.
+      Ory builds open source auth* software because the internet needs a secure
```

Comment:
```suggestion
      Ory builds open source software because the internet needs a secure
```
IMO would be better without the auth, and it is contained in the second part of the sentence as well.